### PR TITLE
Sponsor layout and BG updates

### PR DESCRIPTION
**Pull request by snyderks**
Sponsor images now lay out two to a row
Move down to one per row on small widths
Background color of sponsor page is white

### CODE REVIEW

**Comment by isiah-lloyd** [comment]
Also tagging @zacharysang and @zcollins0 for their feedback. Here's a current screenshot of the branch:
image

**Comment by noahbass** [comment]
I agree. More whitespace would be better. And normalized sizing for logos would also be better.

**Comment by snyderks** [comment]
Changed the image spacing and fixed the large amount of padding at small screen widths

**Comment by noahbass** [comment]
I think this would be good to go if each column was 50% and each image was centered in its column

untitled

**Comment by snyderks** [comment]
fixed layout
This requires a fair bit of finessing because there's a max width on the container but the images have a defined height, which can result in stretching. At the moment it's fine, but if there's an image with a higher aspect ratio than Vantiv's, there might be issues.

File: sponsors.html [hunk 1]
```diff
@@ -7,6 +7,12 @@
 useContainer: false
 ---
 
+<style type="text/css">
```
Comment (noahbass) [inline sponsors.html]: We can move this to a sass file at some point

**Pull request merged**